pkg/gktest/read_constraints.go

```diff
@@ -72,13 +71,10 @@ func readUnstructured(bytes []byte) (*unstructured.Unstructured, error) {
 	return u, nil
 }
 
-// TODO(willbeason): Remove once ToVersionless() is threadsafe.
```

Comment:
Is this threadsafe now to remove the lock?

Reply:
It took two PRs but yes 😆

1. https://github.com/open-policy-agent/frameworks/pull/138
1. https://github.com/open-policy-agent/frameworks/pull/142

Reply:
I've also confirmed that these frameworks changes _are_ in gatekeeper already